docs/features/app-features/readme.mdx

```diff
@@ -138,3 +138,4 @@ Relatedly, an app's content is run through the various software layers of OpenSR
 - Content versioning linked to release versioning
 - Questionnaire widget to capture GPS
 - Background capture of GPS linked to stored with specific actions
+- Load all the patients of a particular family in a questionnaire so as to select the specific patient.
```

Comment:
@AngelaKabari Can you elaborate more on this functionality? How will the patients be represented in the Questionnaire? What information about the Patient do we need to render on the Questionnaire?

Reply:
@ellykits the feature is the ability to load all the patients of a particular family into a questionnaire so that the user can select a specific patient. 

Use cases: Most of OpenSRP2 CHW projects will have the concept of families. There are multiple scenarios where it is necessary to list out the family members in order to:
- Assign them to something such as a vaccine;
- Tying  children to their mothers for post natal care (PNC);
- Indicate a health condition e.g: COVID affected.

Potential solutions include:

- Using answer and variable expressions - use a variable expression to fetch the group data into the questionnaire, then use the [answer expression](https://github.com/ona-health/sid-dto-fhir-resources/blob/5326ce95f8f25b1f575acbe6%5B%E2%80%A6%5Duestionnaire/patient_registration/family_registration.fhir.json) to display the list of the family members. Querying for the patients using the family name e,g. `Patient?family=somename`, will work if the patients have a family name, but won’t work if there is no family name or if 2 families have the same name. This can be circumvented by using `address-postalcode`, assuming every family has one and that it’s unique, then combining the family name + address to generate a unique search parameter. 
- [Reverse chaining](https://www.hl7.org/fhir/search.html#has) with an xfhir query. This works on https://fhir.demo.smartregister.org/fhir/Patient?_has:Group:member:_id=cce041e0-b381-4fd6-91a1-a1ee8d74df02 but the `_has` functionality is not supported on SDK currently.

Let me know if you have any further questions or need further clarification?

Cc @pld

Reply:
OK, this does not sound like a product feature, but a content configuration, i.e. it is a feature the product currently supports. We don't have a developed strategy for managing "repeatable content configuration patterns", or another term we might want to use for this. I could see creating a broader roadmap item here that's around "document and provide examples for common content configuration patterns" and this is an example of a pattern to include in that set. How does that re-framing sound to you?

Reply:
Sounds good to me.

Reply:
cool updated